pkg/descriptions.json

```diff
@@ -566,6 +566,13 @@
     ]
   },
   {
+    "Name": "System.Diagnostics.PerformanceCounter",
```

Comment:
Given you have this, I don't understand why you're getting `
D:\j\workspace\windows-TGrou---0d2c9ac4\Tools\Packaging.targets(895,5): error : Unable to find description for package System.Diagnostics.PerformanceCounters [D:\j\workspace\windows-TGrou---0d2c9ac4\src\System.Diagnostics.PerformanceCounter\pkg\System.Diagnostics.PerformanceCounters.pkgproj]`

@weshaggard ?